apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableFeatureAssertionsSpec.kt

```diff
@@ -0,0 +1,21 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.fun1
+import ch.tutteli.atrium.specs.notImplemented
+import ch.tutteli.atrium.specs.property
+
+class IterableFeatureAssertionsSpec : ch.tutteli.atrium.specs.integration.IterableFeatureAssertionsSpec(
+    property<Iterable<Int>, Int>(Expect<Iterable<Int>>::min),
```

Comment:
@robstoll this property does not actually exist. should I remove it from the parent?

Reply:
Use fun0 instead

Reply:
I have had to use the val to make it work. please check if done correctly.

Reply:
Use a fun and not a val and use fun0 and not property

Reply:
ok

Reply:
but invocation of min() should return `Expect<E>` instead of `Expect<Iterable<E>>` right?

Reply:
That's right

Reply:
so I cant use Fun0, as it has only one type variable

Reply:
The return type whats wrong with it? (See review comment, maybe we don't tslk about the same)